lib/sourceParams.js

```diff
@@ -40,6 +40,7 @@ module.exports = function(app) {
     shutdownTimeout: params.shutdownTimeout || pkg.shutdownTimeout || 30000,
     bodyParserUrlencodedParams: params.bodyParserUrlencodedParams || pkg.rooseveltConfig.bodyParserUrlencodedParams || { extended: true },
     bodyParserJsonParams: params.bodyParserJsonParams || pkg.rooseveltConfig.bodyParserJsonParams || {},
+    nodeEnv: params.nodeEnv || pkg.rooseveltConfig.nodeEnv,
```

Comment:
if neither the param nor the rooseveltConfig supplies `nodeEnv`, it should default to `process.env.NODE_ENV`